corehq/apps/app_manager/templates/app_manager/partials/case_config_shared.html

```diff
@@ -180,3 +182,12 @@
         </a>
     </div>
 </script>
+
+<script type="text/html" id="case-config:refresh-form-questions">
+    <div style="margin-right: 1em;"
```

Comment:
is `style=...` a thing we do frequently? Best practice is to put that in a CSS file, right?